ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/DescribeConnectorExecutorTest.java

```diff
@@ -173,6 +176,7 @@ public void shouldDescribeKnownConnector() {
 
     // Then:
     verify(engine).getMetaStore();
+    verify(engine).metricCollectors();
```

Comment:
it's a bit silly that we have to verify interactions we don't care about, but I've fought enough battles for today.